name = "throw";
id = "throw"
alias[] = ""
description = "Throw is the keyword which raise an existing exception. "
code = "<?php
try{
    throw new Exception('Error');
} catch (\Exception $e) {
    print 'Warning : error was identified!';
}

?>"
documentation = "https://www.php.net/manual/en/language.exceptions.php"
analyzers[] = "Exceptions/ThrowFunctioncall"
analyzers[] = "Php/ThrowWasAnExpression"
analyzers[] = "Php/ThrowUsage"
cobblers[] = ""
phpVersionSince = ""
phpVersionUntil = ""
seeAlso[] = ""
related[] = "try-catch"
related[] = "exception"
